recipes/winpcap/all/conanfile.py

```diff
@@ -0,0 +1,141 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class WinPcapConan(ConanFile):
+    name = "winpcap"
+    description = "For many years, WinPcap has been recognized as the industry-standard tool for link-layer network access in Windows environments, " \
+                  "allowing applications to capture and transmit network packets bypassing the protocol stack, and including kernel-level packet filtering, " \
+                  "a network statistics engine and support for remote packet capture."
+    license = "BSD-3-Clause"
+    topics = ("conan", "packet", "capture", "filtering", "network", "protocol")
+    homepage = "https://www.winpcap.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "capture_type": ["bpf", "dag", "dlpi", "enet", "nit", "null", "libdlpi", "linux", "pf", "sita", "snit", "snoop", "win32"],
+        "remote": [True, False],
+        "bluetooth": [True, False],
+        "usb_sniffing": [True, False],
+        "usb_sniffing_device": "ANY",
+        "turbocap": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "capture_type": "null",
+        "remote": True,
+        "bluetooth": False,
+        "usb_sniffing": True,
+        "usb_sniffing_device": "/dev/usbmon",
+        "turbocap": False,
+    }
+
+    exports_sources = "CMakeLists.txt", "cmake/*", "patches/*"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_os(self):
+        if hasattr(self, "settings_build"):
+            return self.settings_build.os
+        else:
+            return self.settings.os
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            self.options.turbocap = True
+            self.options.capture_type = "win32"
+        elif self.settings.os == "Linux":
```

Comment:
@madebr WinPcap is for Windows only. It's actually a Windows port of [libpcap](https://www.tcpdump.org/) which is not available on Windows. For Linux/macOS the [libpcap Conan package](https://conan.io/center/libpcap) should be used

Reply:
Then I'll look into stripping everything non-Windows related.